deployment.go

```diff
@@ -44,6 +46,14 @@ type DeploymentStep struct {
 	App    string `json:"app"`
 }
 
+// DeploymentStepV1 is a Marathon v1.X-compatible DeploymentStep
+type DeploymentStepV1 struct {
+	Actions []struct {
+		Type string `json:"type"`
```

Comment:
Oh yes, you're absolutely right. Missed to see that the latest change hasn't been released yet. 👍